### PR TITLE
tesxtual fixes - data sources

### PR DESCRIPTION
**Pull request by henriterhofte**
### Checklist

- This textual update was NOT tested.
- 
### Description

The Sources / Bronnen screen is now titled "Data Sources / Databronnen"; in other texts this terminology is consistently maintained. 

### Linked Issues
https://trello.com/c/1XVwOAbK

### Additional context

These are the textual suggestions based on my review of 21 and 22 October. See https://trello.com/c/1XVwOAbK for issues that MUST be resolved and issues that MAY be resolved.

### CODE REVIEW

File: .github/workflows/eas-build.yml [hunk 1]
```diff
@@ -36,7 +36,7 @@ jobs:
           ssh-private-key: ${{ secrets.GH_SSH_KEY }}
 
       - name: Install dependencies
-        run: yarn install --production
+        run: sudo yarn install --production
```
Comment (jortdr) [inline .github/workflows/eas-build.yml:39]: Not sure if this was meant to be changed.

**Review by jortdr — COMMENTED** [review]
Changes seem fine, just one comment on the workflow file. @henriterhofte or @n-vr could you verify and merge when you're ready?

**Comment by jortdr** [comment]
Also, not sure if it's intentional to add all the versioning changes to this as well.

It might have been on accident since this commit was based on main.

We're waiting on this PR to be merged before opening a new PR to main for next release.

File: .github/workflows/eas-build.yml (hunk 1 above)
Comment (n-vr) [reply .github/workflows/eas-build.yml:39]: Yes, intentional.

**Comment by n-vr** [comment]
> Also, not sure if it's intentional to add all the versioning changes to this as well.
> 
> It might have been on accident since this commit was based on main.
> 
> We're waiting on this PR to be merged before opening a new PR to main for next release.

I reverted this in main. I really don't know why the commits from the main branch are shown in this PR...

**Comment by n-vr** [comment]
I am still working on the Github Actions build pipeline, but will no longer commit directly to the main branch. Fix will come in another branch and PR.

**Pull request merged**